documentation/asciidoc/accessories/display/display_intro.adoc

```diff
@@ -1,47 +1,78 @@
 == Raspberry Pi Touch Display
 
-The https://www.raspberrypi.com/products/raspberry-pi-touch-display/[Raspberry Pi Touch Display] is an LCD display which connects to the Raspberry Pi through the DSI connector. In recent versions of Raspberry Pi OS, both the HDMI and LCD display outputs can be used at the same time.
+The https://www.raspberrypi.com/products/raspberry-pi-touch-display/[Raspberry Pi Touch Display] is an LCD display which connects to the Raspberry Pi through the DSI connector. You can also use the HDMI and DSI display outputs at the same time.
 
 === Board Support
```

Comment:
Not sure this requires an entire section to itself. Could be a NOTE block perhaps, or even just leave it inline in the main narrative.